shallow_backup.py

```diff
@@ -657,6 +657,13 @@ def prompt_for_path_update(config):
 		move_git_folder_to_path(current_path, abs_path)
 
 
+def clean_backup_dir(backup_path):
+	try:
+		shutil.rmtree(backup_path)
+		print("{} Deleted backup directory {} {}".format(Fore.BLUE, backup_path, Style.RESET_ALL))
```

Comment:
1. We should use `Fore.RED` and `Style.BRIGHT` here since removal is a "dangerous" action.
2. I usually put print statements before the action, and use the infinitive tense. "Deleting" and then I finish the line with ellipsis.